plaso/storage/sqlite/sqlite_file.py

```diff
@@ -719,13 +722,35 @@ def GetAnalysisReports(self):
     """
     return self._GetAttributeContainers(self._CONTAINER_TYPE_ANALYSIS_REPORT)
 
-  def GetErrors(self):
-    """Retrieves the errors.
+  def GetWarnings(self):
+    """Retrieves the warnings.
 
     Returns:
-      generator(ExtractionError): error generator.
+      generator(ExtractionWarning): warning generator.
+    """
+    # For backwards compatibility with pre-20190309 stores.
+    # Note that stores cannot contain both ExtractionErrors and
+    # ExtractionWarnings
+    if self._HasAttributeContainers(self._CONTAINER_TYPE_EXTRACTION_ERROR):
```

Comment:
What if I open an existing store and add more events and warnings?

Reply:
Done in #2407